src/ecc/codes/gottesman.jl

```diff
@@ -0,0 +1,51 @@
+struct QHamming <: AbstractECC
+    j::Int
+    function QHamming(j)
+        if j < 3
+            error("j must be greater than or equal to 3 in order to create a valid code of tractable size")
+        end
```

Comment:
```suggestion
        (j >= 3 && j < 16) || error("In `QHamming(j)`, `j` must be ≥  3 in order to obtain a valid code and `j` must be < 16 to remain tractable")
```

Your check was only for upper limit, but not for lower limits. I updated it and shortened the syntax

Reply:
Thanks! For the upper bound, QHamming (16) works for j=16 a well!  I added a demonstration of QHamming (20) above! What should be the upper bound in this case? Should I check the cases for latest upper bound?